exchange/auction.go

```diff
@@ -149,11 +148,50 @@ func isNewWinningBid(bid, wbid *openrtb2.Bid, preferDeals bool) bool {
 	return bid.Price > wbid.Price
 }
 
+func (a *auction) validateAndUpdateMultiBid(adapterBids map[openrtb_ext.BidderName]*entities.PbsOrtbSeatBid, preferDeals bool, accountDefaultBidLimit int) {
+	bidsDropped := false
+	// sort bids for multibid targeting
+	for _, topBidsPerBidder := range a.winningBidsByBidder {
+		for bidder, topBids := range topBidsPerBidder {
+			sort.Slice(topBids, func(i, j int) bool {
+				return isNewWinningBid(topBids[i].Bid, topBids[j].Bid, preferDeals)
+			})
+
+			// assert hard limit on bids count per imp, per adapter. ()
+			if accountDefaultBidLimit != 0 && len(topBids) > accountDefaultBidLimit {
+				for i := accountDefaultBidLimit; i < len(topBids); i++ {
+					topBids[i].Bid = nil
+					topBids[i] = nil
+					bidsDropped = true // mark drop-candidate bid and remove its reference (avoid mem leak)
```

Comment:
Nitpick: I think this comment can be removed.

Reply:
done